aql.py

```diff
@@ -45,18 +45,25 @@ def __init__(self, url, name):
     def query(self):
         return Query(self)
 
-    def addVertex(self, id, label, prop={}):
+    def addVertex(self, id, label, prop={}, update=False):
         payload = json.dumps({
             "gid" : id,
             "label" : label,
             "data" : prop
         })
         headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
-        request = urllib2.Request(self.url + "/" + self.name + "/vertex", payload, headers=headers)
+        url = self.url + "/" + self.name + "/vertex"
+        if update:
+            url += "?update=true"
+        print url
```

Comment:
Remove the debug line.

Also, why is this using a URL parameter? Shouldn't it be part of the request document?